test/timed_asset_lock_contract_test.py

```diff
@@ -262,69 +266,17 @@ def test_deploy(self, app_id, client, asset_id, wallet_1, wallet_2, end_time):
         global_state = read_global_state(client, public_key, app_id)
         assert_state(local_state, global_state, asset_id, public_key, end_time)
 
+    def test_4_group(self, client, app_id, wallet_1, asset_id):
+        from akita_inu_asa_utils import (get_application_address, get_asset_balance)
+        from algosdk.future import transaction
+
         # got to fund the contract with algo
         app_public_key = get_application_address(app_id)
-
-        params = client.suggested_params()
-        txn, txn_id = payment_signed_txn(private_key,
-                                         public_key,
-                                         app_public_key, 300000, params)
-        client.send_transactions([txn])
-        wait_for_txn_confirmation(client, txn_id, 5)
-        assert_adversary_actions(app_id, wallet_2, client, asset_id)
-
-        # users fail clearing apps they are not opted in to
-        assert_adversary_actions(app_id, wallet_1, client, asset_id, False, fail_clear=True)
-
-    def test_on_opt_in(self, app_id, wallet_1, wallet_2, client, asset_id, end_time):

```

Comment:
Maybe we should leave single transaction tests just to make sure they’re working individually.

Reply:
The contract has been refactored so that individual trasactions (besides create and delete) do not work